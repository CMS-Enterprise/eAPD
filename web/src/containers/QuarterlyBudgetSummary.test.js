import { shallow } from 'enzyme';
import React from 'react';

import {
  plain as QuarterlyBudgetSummary,
  mapStateToProps
} from './QuarterlyBudgetSummary';

describe('quarterly budget summary component', () => {
  test('renders correctly if data is not loaded', () => {
    const component = shallow(
      <QuarterlyBudgetSummary budget={{}} years={[]} />
    );
    expect(component).toMatchSnapshot();
  });

  test('renders correctly if data is loaded', () => {
    const component = shallow(
      <QuarterlyBudgetSummary
        budget={{
          hitAndHie: {
            '1': {
              1: {
                combined: 1,
                contractors: 2,
                inHouse: 3
              },
              2: {
                combined: 4,
                contractors: 5,
                inHouse: 6
              },
              3: {
                combined: 7,
                contractors: 8,
                inHouse: 9
              },
              4: {
                combined: 10,
                contractors: 11,
                stainHousete: 12
              },
              subtotal: {
                combined: 13,
                contractors: 14,
                inHouse: 15
              }
            },
            '2': {
              1: {
                combined: 101,
                contractors: 102,
                inHouse: 103
              },
              2: {
                combined: 104,
                contractors: 105,
                inHouse: 106
              },
              3: {
                combined: 107,
                contractors: 108,
                inHouse: 109
              },
              4: {
                combined: 110,
                contractors: 111,
                inHouse: 112
              },
              subtotal: {
                combined: 113,
                contractors: 114,
                inHouse: 115
              }
            },
            total: {
              combined: 3000,
              contractors: 3001,
              inHouse: 3002
            }
          },
          mmis: {
            '1': {
              1: {
                combined: 201,
                contractors: 202,
                inHouse: 203
              },
              2: {
                combined: 204,
                contractors: 205,
                inHouse: 206
              },
              3: {
                combined: 207,
                contractors: 208,
                inHouse: 209
              },
              4: {
                combined: 210,
                contractors: 211,
                inHouse: 212
              },
              subtotal: {
                combined: 213,
                contractors: 214,
                inHouse: 215
              }
            },
            '2': {
              1: {
                combined: 1101,
                contractors: 1102,
                inHouse: 1103
              },
              2: {
                combined: 1104,
                contractors: 1105,
                inHouse: 1106
              },
              3: {
                combined: 1107,
                contractors: 1108,
                inHouse: 1109
              },
              4: {
                combined: 1110,
                contractors: 1111,
                inHouse: 1112
              },
              subtotal: {
                combined: 1113,
                contractors: 1114,
                inHouse: 1115
              }
            },
            total: {
              combined: 4000,
              contractors: 4001,
              inHouse: 4002
            }
          }
        }}
        years={['1', '2']}
      />
    );
    expect(component).toMatchSnapshot();
  });

  test('maps state to props', () => {
    const state = {
      apd: { data: { years: 'years' } },
      budget: { federalShareByFFYQuarter: 'federal share' }
    };

    expect(mapStateToProps(state)).toEqual({
      budget: 'federal share',
      years: 'years'
    });
  });
});
