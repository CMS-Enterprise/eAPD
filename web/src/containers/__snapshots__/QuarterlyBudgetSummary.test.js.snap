// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quarterly budget summary component renders correctly if data is loaded 1`] = `
<Fragment>
  <div
    className="mb3"
    key="hitAndHie"
  >
    <h3
      className="ds-h3"
    >
      HIT and HIE
    </h3>
    <table
      className="table-cms"
    >
      <colgroup>
        <col
          className="table-cms--col-header__fixed-width"
        />
        <col />
        <col />
        <col />
        <col />
      </colgroup>
      <thead>
        <tr>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-1"
            key="1"
          >
            FFY 1
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-1-q1"
            key="1"
          >
            Q1
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-1-q2"
            key="2"
          >
            Q2
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-1-q3"
            key="3"
          >
            Q3
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-1-q4"
            key="4"
          >
            Q4
          </th>
          <th
            className="right-align"
            id="quarterly-budget-summary-hitAndHie-fy-1-subtotal"
          >
            Subtotal
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          key="state"
        >
          <th
            headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2"
          >
            In-House Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q1"
            key="1"
          >
            <Dollars>
              3
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q2"
            key="2"
          >
            <Dollars>
              6
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q3"
            key="3"
          >
            <Dollars>
              9
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q4"
            key="4"
          >
            <Dollars>
              12
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-subtotal"
          >
            <Dollars>
              15
            </Dollars>
          </td>
        </tr>
        <tr
          key="contractors"
        >
          <th
            headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2"
          >
            Private Contractor Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q1"
            key="1"
          >
            <Dollars>
              2
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q2"
            key="2"
          >
            <Dollars>
              5
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q3"
            key="3"
          >
            <Dollars>
              8
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q4"
            key="4"
          >
            <Dollars>
              11
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-subtotal"
          >
            <Dollars>
              14
            </Dollars>
          </td>
        </tr>
        <tr
          key="combined"
        >
          <th
            headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2"
          >
            Total Enhanced FFP
          </th>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q1"
            key="1"
          >
            <Dollars>
              1
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q2"
            key="2"
          >
            <Dollars>
              4
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q3"
            key="3"
          >
            <Dollars>
              7
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-q4"
            key="4"
          >
            <Dollars>
              10
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-fy-1 quarterly-budget-summary-hitAndHie-fy-1-subtotal"
          >
            <Dollars>
              13
            </Dollars>
          </td>
        </tr>
      </tbody>
    </table>
    <table
      className="table-cms"
    >
      <colgroup>
        <col
          className="table-cms--col-header__fixed-width"
        />
        <col />
        <col />
        <col />
        <col />
      </colgroup>
      <thead>
        <tr>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-2"
            key="2"
          >
            FFY 2
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-2-q1"
            key="1"
          >
            Q1
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-2-q2"
            key="2"
          >
            Q2
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-2-q3"
            key="3"
          >
            Q3
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-hitAndHie-fy-2-q4"
            key="4"
          >
            Q4
          </th>
          <th
            className="right-align"
            id="quarterly-budget-summary-hitAndHie-fy-2-subtotal"
          >
            Subtotal
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          key="state"
        >
          <th
            headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2"
          >
            In-House Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q1"
            key="1"
          >
            <Dollars>
              103
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q2"
            key="2"
          >
            <Dollars>
              106
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q3"
            key="3"
          >
            <Dollars>
              109
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q4"
            key="4"
          >
            <Dollars>
              112
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-subtotal"
          >
            <Dollars>
              115
            </Dollars>
          </td>
        </tr>
        <tr
          key="contractors"
        >
          <th
            headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2"
          >
            Private Contractor Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q1"
            key="1"
          >
            <Dollars>
              102
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q2"
            key="2"
          >
            <Dollars>
              105
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q3"
            key="3"
          >
            <Dollars>
              108
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q4"
            key="4"
          >
            <Dollars>
              111
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-subtotal"
          >
            <Dollars>
              114
            </Dollars>
          </td>
        </tr>
        <tr
          key="combined"
        >
          <th
            headers="quarterly-budget-summary-hitAndHie-null1 quarterly-budget-summary-hitAndHie-null2"
          >
            Total Enhanced FFP
          </th>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q1"
            key="1"
          >
            <Dollars>
              101
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q2"
            key="2"
          >
            <Dollars>
              104
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q3"
            key="3"
          >
            <Dollars>
              107
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-q4"
            key="4"
          >
            <Dollars>
              110
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-fy-2 quarterly-budget-summary-hitAndHie-fy-2-subtotal"
          >
            <Dollars>
              113
            </Dollars>
          </td>
        </tr>
      </tbody>
    </table>
    <table
      className="table-cms table-cms__totals"
    >
      <colgroup>
        <col
          className="table-cms--col-header__fixed-width"
        />
        <col />
        <col />
        <col />
        <col />
      </colgroup>
      <thead>
        <tr>
          <th
            id="quarterly-budget-summary-hitAndHie-total"
          >
            Total 
            HIT and HIE
          </th>
          <th
            id="quarterly-budget-summary-hitAndHie-null1"
          />
        </tr>
      </thead>
      <tbody>
        <tr>
          <th
            headers="quarterly-budget-summary-hitAndHie-total"
          >
            In-House Costs
          </th>
          <td
            className="bold mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-total2 quarterly-budget-summary-hitAndHie-total"
          >
            <Dollars>
              3002
            </Dollars>
          </td>
        </tr>
        <tr>
          <th
            headers="quarterly-budget-summary-hitAndHie-total"
          >
            Private Contractor Costs
          </th>
          <td
            className="bold mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-total2 quarterly-budget-summary-hitAndHie-total"
          >
            <Dollars>
              3001
            </Dollars>
          </td>
        </tr>
        <tr>
          <th
            headers="quarterly-budget-summary-hitAndHie-total"
          >
            Total Enhanced FFP
          </th>
          <td
            className="bold mono right-align nowrap"
            headers="quarterly-budget-summary-hitAndHie-total2 quarterly-budget-summary-hitAndHie-total"
          >
            <Dollars>
              3000
            </Dollars>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div
    className="mb3"
    key="mmis"
  >
    <h3
      className="ds-h3"
    >
      MMIS
    </h3>
    <table
      className="table-cms"
    >
      <colgroup>
        <col
          className="table-cms--col-header__fixed-width"
        />
        <col />
        <col />
        <col />
        <col />
      </colgroup>
      <thead>
        <tr>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-1"
            key="1"
          >
            FFY 1
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-1-q1"
            key="1"
          >
            Q1
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-1-q2"
            key="2"
          >
            Q2
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-1-q3"
            key="3"
          >
            Q3
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-1-q4"
            key="4"
          >
            Q4
          </th>
          <th
            className="right-align"
            id="quarterly-budget-summary-mmis-fy-1-subtotal"
          >
            Subtotal
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          key="state"
        >
          <th
            headers="quarterly-budget-summary-mmis-null1 quarterly-budget-summary-mmis-null2"
          >
            In-House Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q1"
            key="1"
          >
            <Dollars>
              203
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q2"
            key="2"
          >
            <Dollars>
              206
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q3"
            key="3"
          >
            <Dollars>
              209
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q4"
            key="4"
          >
            <Dollars>
              212
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-subtotal"
          >
            <Dollars>
              215
            </Dollars>
          </td>
        </tr>
        <tr
          key="contractors"
        >
          <th
            headers="quarterly-budget-summary-mmis-null1 quarterly-budget-summary-mmis-null2"
          >
            Private Contractor Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q1"
            key="1"
          >
            <Dollars>
              202
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q2"
            key="2"
          >
            <Dollars>
              205
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q3"
            key="3"
          >
            <Dollars>
              208
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q4"
            key="4"
          >
            <Dollars>
              211
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-subtotal"
          >
            <Dollars>
              214
            </Dollars>
          </td>
        </tr>
        <tr
          key="combined"
        >
          <th
            headers="quarterly-budget-summary-mmis-null1 quarterly-budget-summary-mmis-null2"
          >
            Total Enhanced FFP
          </th>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q1"
            key="1"
          >
            <Dollars>
              201
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q2"
            key="2"
          >
            <Dollars>
              204
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q3"
            key="3"
          >
            <Dollars>
              207
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-q4"
            key="4"
          >
            <Dollars>
              210
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-fy-1 quarterly-budget-summary-mmis-fy-1-subtotal"
          >
            <Dollars>
              213
            </Dollars>
          </td>
        </tr>
      </tbody>
    </table>
    <table
      className="table-cms"
    >
      <colgroup>
        <col
          className="table-cms--col-header__fixed-width"
        />
        <col />
        <col />
        <col />
        <col />
      </colgroup>
      <thead>
        <tr>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-2"
            key="2"
          >
            FFY 2
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-2-q1"
            key="1"
          >
            Q1
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-2-q2"
            key="2"
          >
            Q2
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-2-q3"
            key="3"
          >
            Q3
          </th>
          <th
            className="center"
            id="quarterly-budget-summary-mmis-fy-2-q4"
            key="4"
          >
            Q4
          </th>
          <th
            className="right-align"
            id="quarterly-budget-summary-mmis-fy-2-subtotal"
          >
            Subtotal
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          key="state"
        >
          <th
            headers="quarterly-budget-summary-mmis-null1 quarterly-budget-summary-mmis-null2"
          >
            In-House Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q1"
            key="1"
          >
            <Dollars>
              1103
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q2"
            key="2"
          >
            <Dollars>
              1106
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q3"
            key="3"
          >
            <Dollars>
              1109
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q4"
            key="4"
          >
            <Dollars>
              1112
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-subtotal"
          >
            <Dollars>
              1115
            </Dollars>
          </td>
        </tr>
        <tr
          key="contractors"
        >
          <th
            headers="quarterly-budget-summary-mmis-null1 quarterly-budget-summary-mmis-null2"
          >
            Private Contractor Costs
          </th>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q1"
            key="1"
          >
            <Dollars>
              1102
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q2"
            key="2"
          >
            <Dollars>
              1105
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q3"
            key="3"
          >
            <Dollars>
              1108
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap false"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q4"
            key="4"
          >
            <Dollars>
              1111
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-subtotal"
          >
            <Dollars>
              1114
            </Dollars>
          </td>
        </tr>
        <tr
          key="combined"
        >
          <th
            headers="quarterly-budget-summary-mmis-null1 quarterly-budget-summary-mmis-null2"
          >
            Total Enhanced FFP
          </th>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q1"
            key="1"
          >
            <Dollars>
              1101
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q2"
            key="2"
          >
            <Dollars>
              1104
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q3"
            key="3"
          >
            <Dollars>
              1107
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap true"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-q4"
            key="4"
          >
            <Dollars>
              1110
            </Dollars>
          </td>
          <td
            className="mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-fy-2 quarterly-budget-summary-mmis-fy-2-subtotal"
          >
            <Dollars>
              1113
            </Dollars>
          </td>
        </tr>
      </tbody>
    </table>
    <table
      className="table-cms table-cms__totals"
    >
      <colgroup>
        <col
          className="table-cms--col-header__fixed-width"
        />
        <col />
        <col />
        <col />
        <col />
      </colgroup>
      <thead>
        <tr>
          <th
            id="quarterly-budget-summary-mmis-total"
          >
            Total 
            MMIS
          </th>
          <th
            id="quarterly-budget-summary-mmis-null1"
          />
        </tr>
      </thead>
      <tbody>
        <tr>
          <th
            headers="quarterly-budget-summary-mmis-total"
          >
            In-House Costs
          </th>
          <td
            className="bold mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-total2 quarterly-budget-summary-mmis-total"
          >
            <Dollars>
              4002
            </Dollars>
          </td>
        </tr>
        <tr>
          <th
            headers="quarterly-budget-summary-mmis-total"
          >
            Private Contractor Costs
          </th>
          <td
            className="bold mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-total2 quarterly-budget-summary-mmis-total"
          >
            <Dollars>
              4001
            </Dollars>
          </td>
        </tr>
        <tr>
          <th
            headers="quarterly-budget-summary-mmis-total"
          >
            Total Enhanced FFP
          </th>
          <td
            className="bold mono right-align nowrap"
            headers="quarterly-budget-summary-mmis-total2 quarterly-budget-summary-mmis-total"
          >
            <Dollars>
              4000
            </Dollars>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</Fragment>
`;

exports[`quarterly budget summary component renders correctly if data is not loaded 1`] = `""`;
